ingenialink/dictionary.py

```diff
@@ -433,6 +433,8 @@ class DictionaryV3(Dictionary):
     DEFAULT_ATTR = "default"
     CAT_ID_ATTR = "cat_id"
     UNITS_ATTR = "units"
+    IS_NODE_ID_DEPENDENT_ATTR = "is_node_id_dependent"
```

Comment:
should all these static attributes be public?

Reply:
They are constants, technically they are not protected, but I don't know if we need to add more protections.

Reply:
It's weird if this appears on a doc or autocompletion

Reply:
To make it private, we can add two underscore "__" at the beginning of all the constants. I will create a new issue to do it.